<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Otolith analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1121</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="data">
     <attribute name="title">
      <string>Data input</string>
     </attribute>
     <widget class="QPushButton" name="load_data">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load data</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>50</y>
        <width>231</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Calibration prefixes </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="calibrationPrefixTableWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Sample prefixes </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="samplePrefixTableWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>651</width>
        <height>103</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="load_dir_lineEdit">
         <property name="text">
          <string>/Users/npiro/Documents/ImedeaProject/1. ICPMS_Output</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="load_dir_but">
         <property name="text">
          <string>Select data directory</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="load_drift_file_lineEdit">
         <property name="text">
          <string>/Users/npiro/Documents/ImedeaProject/4. R-Data Calibration/data/Drift_AllSessions_DREAMER.csv</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="load_drift_file_but">
         <property name="text">
          <string>Select drift file</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="load_calibration_file_but">
         <property name="text">
          <string>Select calibration file</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="load_calibration_file_lineEdit">
         <property name="text">
          <string>/Users/npiro/Documents/ImedeaProject/4. R-Data Calibration/data/CRMs_ForR_Abund.csv</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="dataTab">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>1041</width>
        <height>521</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Sessions</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1011</width>
          <height>471</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="30,70">
         <item>
          <widget class="QTreeWidget" name="sessionTreeWidget">
           <column>
            <property name="text">
             <string>Session</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="sessionsDataTableView"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Drift</string>
       </attribute>
       <widget class="QTableView" name="driftDataTableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>1011</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
       <widget class="QTableView" name="calibrationDataTableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>1011</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="labeling">
     <attribute name="title">
      <string>Labeling</string>
     </attribute>
     <widget class="QTabWidget" name="optimizePerSampleSelectionCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1091</width>
        <height>671</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Prediction</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>651</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLineEdit" name="model_path_lineEdit">
           <property name="text">
            <string>/Users/npiro/Documents/ImedeaProject/Python/models/trained</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="select_model_path_but">
           <property name="text">
            <string>Select model file</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTreeWidget" name="labeledSessionTreeWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>181</width>
          <height>261</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Session</string>
         </property>
        </column>
       </widget>
       <widget class="PlotWidget" name="labeledPlotWidget" native="true">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>70</y>
          <width>711</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="label_data_but">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Label data</string>
        </property>
       </widget>
       <widget class="QTreeWidget" name="plotFieldTreeWidget">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>70</y>
          <width>91</width>
          <height>531</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Fields</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="labelElementsTreeWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>340</y>
          <width>181</width>
          <height>251</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Elements</string>
         </property>
        </column>
       </widget>
       <widget class="QCheckBox" name="perSampleSelectionCheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>600</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Optimize selection per sample</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Training</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="calibration">
     <attribute name="title">
      <string>Calibration</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Visualization</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Export</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
